pkg/trace/sampler/exception_sampler.go

```diff
@@ -0,0 +1,210 @@
+package sampler
+
+import (
+	"sync"
+	"sync/atomic"
+	"time"
+
+	"github.com/DataDog/datadog-agent/pkg/trace/metrics"
+	"github.com/DataDog/datadog-agent/pkg/trace/pb"
+	"github.com/DataDog/datadog-agent/pkg/trace/traceutil"
+	"golang.org/x/time/rate"
+)
+
+const (
+	// cardinalityLimit limits the number of spans considered per combination of (env, service).
+	cardinalityLimit = 1000
+	// defaultTTL limits the frequency at which we sample a same span (env, service, name, rsc, ...).
+	defaultTTL = 2 * time.Minute
+	// priorityTTL allows to blacklist p1 spans that are sampled entirely.
+	priorityTTL = 10 * time.Minute
+	// ttlRenewalPeriod prevents from continuously updating expire times for each span seen.
```

Comment:
```suggestion
	// ttlRenewalPeriod specifies the frequency at which we will upload cached entries.
```
I feel the current documentation has no relation to the time value.